src/main/java/com/minecolonies/coremod/proxy/CommonProxy.java

```diff
@@ -91,8 +87,7 @@ public void registerTileEntities()
     @Override
     public void registerEvents()
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1186)